code/Model/Recipient/EmailRecipient.php

```diff
@@ -365,6 +362,18 @@ public function getCMSFields()
             LiteralField::create('EmailPreview', $preview)
         ]);
 
+        $templates = $this->getEmailTemplateDropdownValues();
+
+        if ($templates) {
+            $fields->insertBefore(
+                DropdownField::create(
+                    'EmailTemplate',
+                    _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.EMAILTEMPLATE', 'Email template')
+                )->addExtraClass('toggle-html-only'),
```

Comment:
Think this needs `$templates` somewhere:

```php
            $fields->insertBefore(
                DropdownField::create(
                    'EmailTemplate',
                    _t('SilverStripe\\UserForms\\Model\\UserDefinedForm.EMAILTEMPLATE', 'Email template'),
                   $templates
                )->addExtraClass('toggle-html-only')
```

Reply:
I've updated the PR to include this and hopefully fix the tests.

@wilr FYI I've rebased and force pushed your branch 😃